jenkinsci/jenkins
# [JENKINS-73813] Show a notification when scheduling a build fails

When a build is triggered via the UI (either on the job page or via the buildbuttoncolumn in a view) and the crumb is no longer valid, the tooltip was shown that the build was scheduled, although nothing happened.
This change will show a notification when scheduling failed.

![image](https://github.com/user-attachments/assets/fb655dd2-dade-4122-8148-2118065d8534)



See [JENKINS-73813](https://issues.jenkins.io/browse/JENKINS-73813).



### Testing done
Manual testing
- setup jenkins with users
- create a job without parameters
- go to dashboard
- in a second window logout the user
- click on schedule build -> notification is shown



### Proposed changelog entries

- Show a notification when scheduling a build fails



### Proposed upgrade guidelines

N/A



```[tasklist]
### Submitter checklist
```

### Desired reviewers

@mention



Before the changes are marked as `ready-for-merge`:

```[tasklist]
### Maintainer checklist
```
